InterGalacticCurrencyConvertor/src/main/java/materials/MaterialFactory.java

```diff
@@ -0,0 +1,55 @@
+package main.java.materials;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+/**
+ * Keeps track of different types of materials that can be sold InterGalactically.
+ */
+public final class MaterialFactory {
```

Comment:
Add one line of space between each method. Go and do this in each class.

Reply:
done